main.go

```diff
@@ -97,6 +106,9 @@ func main() {
 		"use the demo provisioner to set host states")
 	flag.StringVar(&healthAddr, "health-addr", ":9440",
 		"The address the health endpoint binds to.")
+	flag.IntVar(&webhookPort, "webhook-port", 9443,
+		"Webhook Server port (set to 0 to disable)")
+	flag.BoolVar(&enableWebhook, "enable-webhook", true, "Enable webhook (default is True)")
```

Comment:
AFAIR, @andfasano suggested it and it made sense to me(flag seems like showing the purpose more clear). But I could not find the conversation in anywhere. I'm fine with both options.

Reply:
It's a user interface, we shouldn't have two ways of doing the same thing that both have to be set in unison.

For readability we can do:

    enableWebhook := webhookPort != 0

Reply:
My main concern here it's just about the readability of the user interface and not the code: I think it's much more clear and easy to understand to have an explicit speaking flag (maybe with a default value bound to an env var, or something like [that](https://book.kubebuilder.io/multiversion-tutorial/webhooks.html)) instead of setting a port value to 0, which could have potentially other meanings to the eyes of the reader.